automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookPyKernelSpec.scala

```diff
@@ -38,48 +38,6 @@ class NotebookPyKernelSpec extends RuntimeFixtureSpec with NotebookTestUtils {
         }
     }
 
-    Seq(Python3).foreach { kernel =>
```

Comment:
I did not realize we tested that 'pip' works! I do hope that pip has some tests internally 😅

Reply:
Sigh 😅